deployment/ansible/roles/driver.web/templates/index.html.j2

```diff
@@ -107,6 +107,7 @@
     <script src="scripts/resources/query-builder-service.js"></script>
     <script src="scripts/resources/aggregate-query-service.js"></script>
     <script src="scripts/resources/saved-filters-service.js"></script>
+    <script src="scripts/resources/blackspots-service.js"></script>
```

Comment:
I think it's worth adding a basic test for this module for coverage reasons/verifying it loads. All of the resource tests are really straightforward, so it'll mostly be a direct copy/paste from one of the others.